public/js/components/App.js

```diff
@@ -35,17 +35,26 @@ const App = React.createClass({
   componentDidMount() {
     this.shortcuts = new KeyShortcuts({ window });
     this.shortcuts.on("Cmd+P", this.toggleSourcesSearch);
+    window.addEventListener("keydown", this.onKeyDown);
```

Comment:
It's weird, since it doesn't seem to support "Escape" also on firefox. It's looking for `window.KeyboardEvent[key]` which doesn't exist for Escape (and also for the other electron mappings AFAICT)